src/app/core/layouts/footer/footer.component.html

```diff
@@ -0,0 +1,49 @@
+<footer>
+  <div class="footer-container">
+    <div class="footer-grid">
+      <div class="footer-section" *ngFor="let section of footerInternalSections">
+        <h3>{{ section.title | translate }}</h3>
+        <div class="footer-links">
+          <ng-container *ngFor="let link of section.links">
+            <a [routerLink]="link.url">{{ link.label | translate }}</a>
+          </ng-container>
+        </div>
+      </div>
+      <div class="footer-section" *ngFor="let section of footerExternalSections">
+        <h3>{{ section.title | translate }}</h3>
+        <div class="footer-social-links">
+          <ng-container *ngFor="let link of section.links">
+            <a [href]="link.url" target="_blank">
+              <span *ngIf="link.label === 'Facebook'">
+                <svg width="24" height="24" viewBox="0 0 24 24">
```

Comment:
You should change the way you insert your SVG's, maybe it should be better to save it as a SVG file (.svg) and use a variable on the .ts file and use it inside an <img src>, because this will really bloat up the code.

Apply the same for other svg's.